extension.neon

```diff
@@ -17,3 +17,10 @@ services:
 			configurationFileLoader: @returnTypesForNewMethodsRuleConfigurationFileLoader
 		tags:
 			- phpstan.rules.rule
+
+    UseTypeHintForNewMethodsRule:
```

Comment:
You shouldn't mix spaces and tabs in the same file. I would prefer to use spaces only if yo'ure ok :)

Reply:
All fixed in https://github.com/PrestaShop/phpstan-prestashop/pull/11/commits/472527f15dd4cdc443bde48ff05f62a899f2973e